js/main.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+var NUMBERS_OF_PHOTOS = 25;
+var LIKES = {
+  min: 15,
+  max: 200
+};
+var NUMBERS = {
+  min: 1,
+  max: 6
+};
+
+var messages = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'
+];
+var names = [
+  'Андрей',
+  'Виктор',
+  'Николай',
+  'Евгений',
+  'Дмитрий',
+  'Александр'
+];
+// Вычисляем случайное число
+var getRandomNumbers = function (min, max) {
+  var randomNumber = Math.floor((Math.random() * (max - min) + min));
+  return randomNumber;
+};
+// Создаем случайные сообщения
+var getRandomTextMessage = function () {
+  var randomBoolean = (Math.floor(Math.random() * 2) === 0);
+  var firstMessage = messages[getRandomNumbers(NUMBERS.min, NUMBERS.max)] + ' ' + messages[getRandomNumbers(NUMBERS.min, NUMBERS.max)];
+  var secondMessage = messages[getRandomNumbers(NUMBERS.min, NUMBERS.max)];
+
+  if (randomBoolean === true) {
```

Comment:
```suggestion
  if (randomBoolean) {
```
Очень странное наименование, из названия переменной непонятно зачем она нужна